### ISSUE TITLE
POD status not updating properly

### ISSUE THREAD

**Issue by hodrigohamalho** [2015-Middleware-Keynote/hexboard#30]
Hexboard and sketchpod
![screen shot 2016-01-08 at 3 38 22 pm](https://cloud.githubusercontent.com/assets/252998/12204840/e9d3a1f4-b61d-11e5-9d73-1124b6f4b110.png)

The pod sketchpod-1-sihq9 is on live state:
![screen shot 2016-01-08 at 3 35 49 pm](https://cloud.githubusercontent.com/assets/252998/12204851/fe375366-b61d-11e5-9bfc-c657de418fe7.png)

Checkin this pod (sketchpod-1-sihq9 ), it seems to be responding
![screen shot 2016-01-08 at 3 36 17 pm](https://cloud.githubusercontent.com/assets/252998/12204874/31d793e8-b61e-11e5-84fa-a938899bc81c.png)
![screen shot 2016-01-08 at 3 37 11 pm](https://cloud.githubusercontent.com/assets/252998/12204875/3596c5e4-b61e-11e5-86af-4ddfca3505a4.png)

**Comment by ryanj** [comment]
What method did you use to set up OpenShift?  

How did you set up the app?

I'm not currently getting this issue in my 3.1 environments, but I've heard a few other people report similar issues.

**Comment by hodrigohamalho** [comment]
I used "quick install" 
```
atomic-openshift-installer install

**Comment by detiber** [comment]
I can confirm this...  From what I could gather it looks like the initial list/watch is working successfully (anything that completes within that timeframe continues to work), but subsequent watches do not appear to be occuring.

**Comment by thoraxe** [comment]
I can confirm this is still kinda happening. I scaled 1->10 and then 10->63 and was left with 2 pods that were running but did not show on the hexboard. I quickly scanned the hexboard log and found no errors.

**Comment by ryanj** [comment]
Thanks for testing.  A few more patches have been merged.

Let me know if you're still seeing the same issues.

**Comment by hodrigohamalho** [comment]
Still not working here.

**Comment by ryanj** [comment]
I'm getting better results with the origin-1.1.3 vm from [openshift.org/vm](https://www.openshift.org/vm/).  [Auto-recovery](https://github.com/2015-Middleware-Keynote/hexboard#auto-recovery) seems to be working, but after deleting a bunch of pods and scaling down to zero, I ended up with a few leftovers:
![hexboard-leftovers](https://cloud.githubusercontent.com/assets/17562/13336160/80c79cca-dbc9-11e5-9249-7c63b9eee3ac.png)

I think the replacement pods are being layered on top of other active pod positions, instead of being rescheduled into available empty slots (which may be leaving orphaned pod statuses on the board after the scale down)

**Comment by detiber** [comment]
Yes, I saw a similar thing when Burr and I presented at DevNexus.

**Comment by hodrigohamalho** [comment]
it's ok on OSE 3.1.6 :)

**Comment by ryanj** [comment]
This seems to be working just fine in the [Origin v1.1.4 VM](https://openshift.org/vm)

**Comment by ryanj** [comment]
Not working in my current OSE environment :(

**Workaround:** Instead of showing autorecovery on the hexboard, show the OpenShift web console instead. It looks good there using both of the available view modes on your project's Overview page (list, topographical)
